doc/client_lua_api.md

```diff
@@ -131,7 +131,7 @@ The `:` prefix can also be used for maintaining backwards compatibility.
 
 Sounds
 ------
-**NOTE: Not fully implemented yet.**
+**NOTE: max_hear_distance and connecting to objects is not implemented.**
```

Comment:
That's how it works for server-sided sounds currently so sounds that move along with the object will need to be implemented separately. I didn't remove the section because it should be added once there is an object ref on the client.